lib/read_parsers.cc

```diff
@@ -41,6 +41,7 @@ SeqAnParser::SeqAnParser( char const * filename ) : IParser( )
     }
     __asm__ __volatile__ ("" ::: "memory");
     _private->seqan_spin_lock = 0;
+    _num_reads = 0;
```

Comment:
Ah ha! Thanks for that S/O post, i finally get it :)

I'll make the change now.